### PR TITLE
Add coveralls.io integration

### LINKED ISSUE #2: Add codecov.io integration
Add codecov.io to show and inspect code coverage.

### PR DESCRIPTION
**Pull request by rubickcz**
Adds coveralls.io integration and improves tests for 100% code coverage.

Resolves #2

### CODE REVIEW

File: requirements_dev.txt [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
+python-coveralls==2.9.1
+sphinx==1.8.1
+sphinx_rtd_theme==0.4.3
 tox==3.5.2
-coverage==4.5.1
-Sphinx==1.8.1
 twine==1.12.1
-sphinx_rtd_theme==0.4.3
+watchdog==0.9.0
+wheel==0.32.1
```
Comment (rubickcz) [inline requirements_dev.txt]: I have added `python-coveralls` and ordered the file alphabetically.

File: requirements_dev.txt [hunk 2]
```diff
@@ -1,10 +1,11 @@
-pip==18.1
 bumpversion==0.5.3
-wheel==0.32.1
-watchdog==0.9.0
+coverage==4.5.1
 flake8==3.6.0
+pip==18.1
```
Comment (radimsuckr) [inline requirements_dev.txt]: Why is there a fixed version for `pip`?

File: requirements_dev.txt (hunk 1 above)
Comment (radimsuckr) [inline requirements_dev.txt]: Same question about the fix version.

File: requirements_dev.txt (hunk 2 above)
Comment (rubickcz) [reply requirements_dev.txt]: This is actually part of the cookie cutter library template. IMO if all packages were without version, it is more likely that errors will arise often.

**Pull request merged**